proj/scalafix.conf

```diff
@@ -1,30 +1,17 @@
-// https://github.com/scalacommunitybuild/scalafix.git#community-build-2.13  # was master; was 2447708096981e179f63fb5648eb67f190e9196f
+// https://github.com/SethTisue/scalafix.git#avoid-arraycharsequence  # was scalacenter, master
 
-// frozen (October 2020); newer commit caused breakage downstream
-// (in simulacrum-scalafix-more)
-
-// forked (November 2020) from 24477080, to adapt to
-// scala/scala#9292 (CharSequence). submitted upstream:
+// temporarily forked pending merge of
 // https://github.com/scalacenter/scalafix/pull/1279
 
 vars.proj.scalafix: ${vars.base} {
   name: "scalafix"
-  uri: "https://github.com/scalacommunitybuild/scalafix.git#ae44ac75960f1b986665066498d18f37b31d38b3"
+  uri: "https://github.com/SethTisue/scalafix.git#cce54f38507ebe70eb847bf202c62290b4ae0724"
 
-  extra.exclude: [
-    // out of scope
-    "docs"
-    // it would really be better if we could build and run the tests, but this
-    // subproject is defined in a really peculiar way that confuses dbuild.
-    // the build overrides compileInputs and resourceGenerators to refer
-    // to the source directories from other subprojects... I don't understand it.
-    // for now anyway, giving up.
-    "unit"
-  ]
-  // testkit wants ScalaTest 3.0
+  extra.exclude: ["docs"]
+  // testkit needs ScalaTest 3.0, but unit needs 3.2. weird...
   deps.inject: ${vars.base.deps.inject} ["scalacommunitybuild#scalatest"]
   extra.commands: ${vars.default-commands} [
-    "removeDependency org.scalatest scalatest"
+    """set testkit / libraryDependencies ~= (_.filterNot(_.toString.contains("scalatest")))"""
```

Comment:
I am not familiar with the community build and dbuild, but from what I understand you also need to patch the scalatest version for the `unit` project which is downstream of `testkit`?

Reply:
yeah — but dbuild's already doing that. everything compiles, it's just that there are problems at runtime. I'll have to dig deeper — it might be as simple as needing to swap the order in which 3.0 and 3.2 appear on the runtime classpath

Reply:
ah, so it's expected that both appear in the runtime classpath? shouldn't sbt evict 3.0?

Reply:
well, it's complicated in a dbuild-specific way. in order for 3.0 and 3.2 to even coexist in dbuild, we had to repackage 3.0 under a different org — that's what the `scalacommunitybuild#scalatest` thing is. since the artifact coordinates are different, eviction doesn't happen. regardless, I should be able to use some sbt-fu and/or dbuild-fu to manually evict things as needed. I'll keep tinkering